patches/server/0809-Boats-do-not-weigh-that-much.patch

```diff
@@ -0,0 +1,33 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Spottedleaf <Spottedleaf@users.noreply.github.com>
+Date: Thu, 12 Aug 2021 23:53:55 -0700
+Subject: [PATCH] Boats do not weigh that much
+
+
+diff --git a/src/main/java/net/minecraft/world/entity/vehicle/Boat.java b/src/main/java/net/minecraft/world/entity/vehicle/Boat.java
+index 99124b70d82140b108d424a5206657efe94f184e..391454a58d18d7373b974e094fd62514ca0d0b6b 100644
+--- a/src/main/java/net/minecraft/world/entity/vehicle/Boat.java
++++ b/src/main/java/net/minecraft/world/entity/vehicle/Boat.java
+@@ -688,7 +688,8 @@ public class Boat extends Entity {
+         this.invFriction = 0.05F;
+         if (this.oldStatus == Boat.Status.IN_AIR && this.status != Boat.Status.IN_AIR && this.status != Boat.Status.ON_LAND) {
+             this.waterLevel = this.getY(1.0D);
+-            this.setDeltaMovement(this.getDeltaMovement().multiply(1.0D, 0.0D, 1.0D).add(0.0, ((double) (this.getWaterLevelAbove() - this.getBbHeight()) + 0.101D) - this.getY(), 0.0)); // Paper
++            this.move(MoverType.SELF, new Vec3(0.0, ((double) (this.getWaterLevelAbove() - this.getBbHeight()) + 0.101D) - this.getY(), 0.0)); // Paper
++            this.setDeltaMovement(this.getDeltaMovement().multiply(1.0D, 0.0D, 1.0D)); // Paper
+             this.lastYd = 0.0D;
+             this.status = Boat.Status.IN_WATER;
+         } else {
+diff --git a/src/main/java/net/minecraft/world/item/BoatItem.java b/src/main/java/net/minecraft/world/item/BoatItem.java
+index 9a11248b13d231c1797e14f843cb8cbec0d35a6e..8998f6233a3135fda928469ae8bb7119e16eee1a 100644
+--- a/src/main/java/net/minecraft/world/item/BoatItem.java
++++ b/src/main/java/net/minecraft/world/item/BoatItem.java
+@@ -67,7 +67,7 @@ public class BoatItem extends Item {
+ 
+                 entityboat.setType(this.type);
+                 entityboat.setYRot(user.getYRot());
+-                if (!world.noCollision(entityboat, entityboat.getBoundingBox().inflate(-net.minecraft.Util.COLLISION_EPSILON))) { // Paper
++                if (!world.noCollision(entityboat, entityboat.getBoundingBox())) { // Paper
+                     return InteractionResultHolder.fail(itemstack);
```

Comment:
Are patches like these going to be merged into the paper equivelants? As it seems this just modifies a paper patch.
https://github.com/PaperMC/Paper/blob/8d182e8639a7d13d4ba8cade2c1532e0f0ef678c/patches/server/0733-Improve-boat-collision-performance.patch